backend/tests/test_sort.py

```diff
@@ -35,3 +35,21 @@ def test_release_date_descending(self):
         hit_release_dates = [hit["release_date"] for hit in search_results["hits"]]
 
         assert sorted(hit_release_dates, reverse=True) == hit_release_dates
+
```

Comment:
A `test_filter.py` test for the imdb_rating would be nice as well

Assert `>= 5.9` gets 5.9 and the one above or something like that